AppCenterCrashes/AppCenterCrashes/MSCrashes.mm

```diff
@@ -274,6 +274,14 @@ + (void)setDelegate:(_Nullable id<MSCrashesDelegate>)delegate {
 
 #pragma mark - Service initialization
 
++ (void)load {
+  [MSAppCenterUserDefaults addKeysToMigrate:@{
+    @"kMSCrashesIsEnabledKey" : @"MSAppCenterCrashesIsEnabledKey",              // MSCrashes
```

Comment:
Are these `// MSCrashes` comments useful?

Reply:
That's the format I adopter by [this comment](https://github.com/microsoft/appcenter-sdk-apple/pull/2013#discussion_r398552143)

Reply:
Makes not much sense here, but in other places points to the place where it's used

Reply:
Got it thanks.